### ISSUE TITLE
A11y: Line item - Collapsing options element lacks focusing

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5064]
The collapsing options element (represented by a down arrow) does not receive focus when navigating via the keyboard. While it is possible to expand the options group, the lack of focus makes it unclear to users that the element is expandable, reducing accessibility for keyboard navigation. (Chrome)
 [https://www.ibm.com/able/toolkit/verify/manual/|https://www.ibm.com/able/toolkit/verify/manual/|smart-link] 

*P.S.* Same issue appears in all pages where custom product line item is available. Please check the task list below.

In Safari, it shows exactly the navigation point, so it is not an issue for keyboard users.

h5. Make sure that...

* Initial focus is evident
* There is a visual indicator every time a key press/action changes focus
* A focused item is differentiated from a selected item

!Screen Recording 2024-10-10 at 09.57.05.mov|width=3826,height=1858,alt="Screen Recording 2024-10-10 at 09.57.05.mov"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-742**



[created from CUS-742, comment 502208]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from CUS-742, comment 511051]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from CUS-742, comment 511052]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from CUS-742, comment 511988]
